subworkflows/local/amp.nf

```diff
@@ -104,30 +105,40 @@ workflow AMP {
         .groupTuple()
         .join( ch_faa_for_ampcombi )
         .join( ch_gbk_for_ampcombi )
+        .join( ch_interpro_for_ampcombi )
         .multiMap{
             input: [ it[0], it[1] ]
             faa: it[2]
             gbk: it[3]
+            interpro: it [4]
         }
 
+    // AMPCOMBI2::PARSETABLES
     if ( params.amp_ampcombi_db != null ) {
-        AMPCOMBI2_PARSETABLES ( ch_input_for_ampcombi.input, ch_input_for_ampcombi.faa, ch_input_for_ampcombi.gbk, params.amp_ampcombi_db_id, params.amp_ampcombi_db, [] )
-    } else {
-        AMP_DATABASE_DOWNLOAD( params.amp_ampcombi_db_id )
-        ch_versions = ch_versions.mix( AMP_DATABASE_DOWNLOAD.out.versions )
-        ch_ampcombi_input_db = AMP_DATABASE_DOWNLOAD.out.db
-        AMPCOMBI2_PARSETABLES ( ch_input_for_ampcombi.input, ch_input_for_ampcombi.faa, ch_input_for_ampcombi.gbk, params.amp_ampcombi_db_id, ch_ampcombi_input_db, [] )
-    }
+        AMPCOMBI2_PARSETABLES ( ch_input_for_ampcombi.input,  ch_input_for_ampcombi.faa,  ch_input_for_ampcombi.gbk, params.amp_ampcombi_db_id, params.amp_ampcombi_db, ch_input_for_ampcombi.interpro )
+        } else {
+            AMP_DATABASE_DOWNLOAD( params.amp_ampcombi_db_id )
+            ch_versions = ch_versions.mix( AMP_DATABASE_DOWNLOAD.out.versions )
+            ch_ampcombi_input_db = AMP_DATABASE_DOWNLOAD.out.db
+            AMPCOMBI2_PARSETABLES ( ch_input_for_ampcombi.input, ch_input_for_ampcombi.faa, ch_input_for_ampcombi.gbk, params.amp_ampcombi_db_id, ch_ampcombi_input_db, ch_input_for_ampcombi.interpro )
+        }
     ch_versions = ch_versions.mix( AMPCOMBI2_PARSETABLES.out.versions )
 
     ch_ampcombi_summaries = AMPCOMBI2_PARSETABLES.out.tsv.map{ it[1] }.collect()
 
-    AMPCOMBI2_COMPLETE ( ch_ampcombi_summaries )
-    ch_versions = ch_versions.mix( AMPCOMBI2_COMPLETE.out.versions )
+    // AMPCOMBI2::PARSETABLES
+    ch_summary_count = ch_ampcombi_summaries.map { it.size() }.sum()
 
-    ch_ampcombi_complete = AMPCOMBI2_COMPLETE.out.tsv
+    if ( ch_summary_count == 0 || ch_summary_count == 1 )  {
+        log.warn("[nf-core/funcscan] AMPCOMBI2: ${ch_summary_count} file passed. Skipping AMPCOMBI2_COMPLETE, AMPCOMBI2_CLUSTER, and TAXONOMY MERGING steps.")
```

Comment:
```suggestion
        log.warn("[nf-core/funcscan] AMPCOMBI2: ${ch_summary_count} file(s) passed. Skipping AMPCOMBI2_COMPLETE, AMPCOMBI2_CLUSTER, and TAXONOMY MERGING steps.")
```

Reply:
It might be good (if not already) to describe in the output file that in some cases you may not get these output files if XYZ